subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DependencyUnresolvedModuleIntegrationTest.groovy

```diff
@@ -140,7 +140,7 @@ class DependencyUnresolvedModuleIntegrationTest extends AbstractHttpDependencyRe
         protocol << ['http', 'https']
     }
 
-    def "blacklists repository from later resolution within the same build on HTTP timeout "() {
+    def "disables repository from later resolution within the same build on HTTP timeout "() {
```

Comment:
How about:

```suggestion
    def "prevents using repository in later resolution within the same build on HTTP timeout"() {
```

(What's the trailing space do in the method name?)

Reply:
+1